tests/functional/object/mcg/test_s3_routes.py

```diff
@@ -146,18 +146,18 @@ def revert_lb_service(self, request):
         """
 
         def finalizer():
-            noobaa_ocp_obj = ocp.OCP(
-                kind="noobaa",
+            storagecluster_obj = ocp.OCP(
+                kind=constants.STORAGECLUSTER,
                 namespace=config.ENV_DATA["cluster_namespace"],
-                resource_name="noobaa",
+                resource_name=constants.DEFAULT_CLUSTERNAME,
             )
             try:
-                if noobaa_ocp_obj.data["spec"]["disableLoadBalancerService"]:
-                    lb_param = (
-                        '[{"op": "remove", "path": "/spec/disableLoadBalancerService"}]'
-                    )
+                if storagecluster_obj.data["spec"]["multiCloudGateway"][
```

Comment:
do you want to get KeyError if the StorageCluster does not have `multiCloudGateway` and nested `disableLoadBalancerService` ? if not we need to use .get()

Reply:
Getting a `KeyError` here from either would be ok since both would mean that `disableLoadBalancerService` is gone. However, we're expecting `storagecluster_obj.data["spec"]["multiCloudGateway"]` not to throw a KeyError since OCS-CI patches some nested properties to it during/right after deployment.